### PR TITLE
Redirect logging to log4j

### PR DESCRIPTION
**Pull request by ilyin371**
- redirect JUL to log4j through SLF4J bridge
- support of -log command line arg with log file size limit = 10MB
- log to console if no log4j.properties provided
- added a wrapper over stock GridLauncher

### CODE REVIEW

File: selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        for (int i = 0; i < args.length; i++) {
+            if (name.equals(args[i])) {
+                index = i;
+                break;
+            }
+        }
+        if (index == -1) {
+            throw new GridConfigurationException("The parameter " + name + " isn't specified.");
+        }
+        if (((index + 1) < args.length) && !args[index + 1].startsWith("-")) {
```
Comment (volkovs) [inline selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java]: Could you please test this method?

File: selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+
+class Logging {
+
+    private static final org.slf4j.Logger LOG = LoggerFactory.getLogger(Logging.class);
+
+    private static final String LOG_FILE_PARAM = "-log";
+    private static final String LOG_FILE_PROPERTY = "selenium.LOGGER";
+
+    private static final String PATTERN = "%d{ISO8601} [%t] [%p] [%c] %m%n";
```
Comment (volkovs) [inline selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java]: Will you add thread Id and User (according to acceptance criteria)?

File: selenium-log4j/src/main/java/io.sterodium.extensions/logging/GridStarter.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+
+public final class GridStarter {
+
+    private GridStarter() {
+    }
+
+    public static void main(String[] args) throws Exception {
+        Logging logging = new Logging(args);
+        logging.configure();
```
Comment (volkovs) [inline selenium-log4j/src/main/java/io.sterodium.extensions/logging/GridStarter.java]: Can't we add it to GridLauncher? Why do we need another wrapper nobody is aware of?

File: selenium-log4j/pom.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            <artifactId>hamcrest-all</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
```
Comment (volkovs) [inline selenium-log4j/pom.xml:65]: Why do you need it?

File: selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java (hunk 1 above)
Comment (ilyin371) [inline selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java]: Done

File: selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java (hunk 2 above)
Comment (ilyin371) [inline selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java]: Thread name is logged (appears to be unique). User object is not applicable here.

File: selenium-log4j/src/main/java/io.sterodium.extensions/logging/GridStarter.java (hunk 3 above)
Comment (ilyin371) [inline selenium-log4j/src/main/java/io.sterodium.extensions/logging/GridStarter.java]: Wrapper is needed to:
1. Configure logging in code.
2. Modify command line arguments that are passed to stock GridLauncher.

**Comment by echoAlexey** [comment]
closes #7

**Pull request merged**